scalableminds/webknossos
# Sort mags before streaming them via zarr streaming

In some datasets / `datasource-properties.json` files the mags (resolutions)  are not sorted correctly. This leads to wk streaming them in this unordered version. This  PR ensures that wk streams the mags in ascending sorted order.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Edit your favourites dataset's datasource-properties.json to have unsorted resolutions / mags for at least on layer
- open the datasource-properties.json via zarr streaming: e.g. localhost:9000/data/zarr3_experimental/sample_organization/l4_sample/
- check the order in which the mags are listed for the layer with unordered resolutions: e.g. localhost:9000/data/zarr3_experimental/sample_organization/l4_sample/color; They should be sorted now. => that should be the fix. 

Compare with broken state via
- https://deploy-preview-36--ome-ngff-validator.netlify.app/?source=https://demo.data2-brain.esc.rzg.mpg.de/data/zarr3_experimental/MPI_Brain_Research/HS_st218_MT3/color
- https://demo.data2-brain.esc.rzg.mpg.de/data/zarr3_experimental/MPI_Brain_Research/HS_st218_MT3/color

### Issues:
- fixes [#8120](https://github.com/scalableminds/webknossos/issues/8120)

------
(Please delete unneeded items, merge only when none are left open)
